meson.build

```diff
@@ -1942,7 +1942,6 @@ if conf.get('HAVE_BLKID') == 1
         executable(
                 'systemd-gpt-auto-generator',
                 'src/gpt-auto-generator/gpt-auto-generator.c',
-                'src/shared/blkid-util.h',
```

Comment:
I was wondering why they were there in the first place... d284b82b3edcfa610c72e66444ff969396be1e3b moved those files from src/basic to src/shared, and the reference here stopped making sense. Before that, they were added in 5c23128daba7236a6080383b2a5649033cfef85c, i.e. the initial conversion to meson, which means that they were inherited from Makefile file lists. It seems OK to remove them.

Reply:
Thank you for digging the log.